autogen/main/variables_defaults.tf.tmpl

```diff
@@ -114,5 +114,14 @@ locals {
     ),
     var.node_pools_linux_node_configs_sysctls
   )
+   node_pools_cgroup_mode = merge(
+    { all = "" },
```

Comment:
@apeabody i wonder if this needs to be `CGROUP_MODE_UNSPECIFIED` to make it work

Reply:
Hi @DrFaust92 - I checked the provider code and `cgroup_mode` is an (optional) `string`, but will not accept `""`.  So an option for https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/2001/files#diff-a3a07be3819af6f1a65ed243a613e69b544b7274d248a4b4d7d4d127aec40aeeR1045 would be to check the locals value and if `""` instead pass 'null'.